Description: Port to the LV2's first stable spec.
Author: Alessio Treglia <alessio@debian.org>
Applied-Upstream: yes
---
 Makefile |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- so-synth-lv2.orig/Makefile
+++ so-synth-lv2/Makefile
@@ -2,7 +2,7 @@ OBJECTS = so-666.o so-kl5.o so-404.o sos
 LIBRARY = libsosynth.so
 TTLS = so-666.ttl so-kl5.ttl so-404.ttl manifest.ttl
 CC = gcc
-CFLAGS += -Wall -O3 -ffast-math -lm `pkg-config --cflags --libs lv2core` -fPIC
+CFLAGS += -Wall -O3 -ffast-math -lm `pkg-config --cflags --libs lv2` -fPIC
 INSTALLDIR = $(DESTDIR)/usr/lib/lv2/
 INSTALLNAME = so-synth.lv2/
 $(LIBRARY) : $(OBJECTS)
